nemo_curator/datasets/doc_dataset.py

```diff
@@ -35,7 +36,7 @@ def __len__(self):
     def persist(self):
         return DocumentDataset(self.df.persist())
 
-    def head(self, n=5):
+    def head(self, n: int = 5):
```

Comment:
```suggestion
    def head(self, n: int = 5) -> ??:
```
since we're adding the annotation let's add the return type too